types/src/account_state.rs

```diff
@@ -37,17 +32,17 @@ impl AccountState {
             .map(|opt_ar| opt_ar.map(|ar| ar.address()))
     }
 
-    pub fn get_diem_account_resource(&self) -> Result<Option<DiemAccountResource>> {
-        self.get_resource::<DiemAccountResource>()
+    pub fn get_diem_account_resource(&self) -> Result<Option<AptosAccountResource>> {
```

Comment:
we probably can remove this function?

Reply:
next pr we can delete even more legacy stuff